README.md

```diff
@@ -31,47 +29,39 @@ $ git submodule update
 ```
 ## Compilation:
 **1** Install 3d-parties developers libraries
+- Run the following commands :
 ```
-sudo apt-get install liblua5.2-dev libxml2-dev lua-lpeg-dev
-sudo apt-get install openssl
-```
-
-**2** Install Qt5.3+
-
-[Download Qt5.3+](https://download.qt.io/archive/qt/5.3/5.3.1/)
-```
-sudo chmod +x some-qt-build.run
-./some-qt-build.run
+$ sudo apt-get install liblua5.2-dev libxml2-dev lua-lpeg-dev
+$ sudo apt-get install openssl
 ```
 
-**2** Setup QMAKE environment variable to path to qmake
-```export QMAKE=${PATH_TO_QMAKE} ```
-*You can get path to qmake this way:*
+**2** Install Qt5.9+
+- Run the following commands :
 ```
-$ sudo find / -name qmake
-/usr/bin/qmake
-/opt/Qt5.3.1/5.3/gcc/bin/qmake
+$ sudo add-apt-repository -y ppa:beineri/opt-qt591-xenial
+$ sudo apt-get update
+$ sudo apt-get install qt59base qt59websockets
 ```
-/usr/bin/qmake in most cases does not work as it is just soft link to qtchooser
+**3** Install cmake 3.11+
+[CMake](https://cmake.org/download/) : Download and install any release version > 3.11
 
-**2**  ```$ make```
+**4** Build ATF
+- Create build directory and get into created directory
+- Run the following command : `cmake <path to sources>`
+- Run `make`
 
- 1) If during executing "make" command you have the following problem:
-   Project ERROR: Unknown module(s) in QT: websockets
-
-   Solution:
-   You have to change location of qmake in Makefile in atf root directory
-   Find location of qmake executable on your local PC:
-   (It should look like: .../Qt/5.3/gcc_64/bin/qmake)
-
-   and put it into Makefile to the line:
-   QMAKE=<your path to qmake>
-   Sometimes you will need reinstall QT Creator to get correct qmake executable
-   Also you can open QT Creator. Then go to: Tool->Options->Build & Run. Find Qt Versions Tab.
-   Here you can find qmake location.
+## Configuration of ATF
+ATF configuration is setting up in `modules/configuration` folder.
+`base_config.lua` - base configuration parameters (reporting, paths to SDL e.t.c)
```

Comment:
This section doesn't show up as expected in https://github.com/smartdevicelink/sdl_atf/tree/feature/remote_atf_connection#configuration-of-atf. Each of these config files should probably be in its own bullet point
```suggestion
- `base_config.lua`: base configuration parameters (reporting, paths to SDL e.t.c)
```

Reply:
@ShobhitAd 
Fixed in:https://github.com/smartdevicelink/sdl_atf/pull/180/commits/bf96531c231b571f8326ea5121ca50677d94700a